src/actions/deriveAddresses.ts

```diff
@@ -221,8 +243,10 @@ function deriveAddress(
  */
 
 function getDerivationMode(address: string) {
-  if (address.match("^(bc1|tb1|ltc1).*")) {
+  if (address.match("^(bc1q|tb1|ltc1).*")) {
```

Comment:
Nice catch! Took me a moment to understand that change.

Reply:
same change for tb probably is needed now want to make it when will have time to create taproot testnet account with transactions for testing